### ISSUE TITLE
knowledge transfer improvements

### ISSUE THREAD

**Issue by FylyppS** [FylyppS/NLP_fakenewsdetection#22]
Analyze the fake-news-nlp\REANALIZA_TRANSFERU_Z_WYMOGIEM_PROJEKTU.md document for context and directions, 
I have already tried to implement the Discriminative LR for scheduler and using a new optimizer to refresh learning rates
it should be focused on fake-news-nlp\improved_train_test_pipeline.py since its used for all the training but feel free to make new files within reason if needed
